models/test/_package.yml

```diff
@@ -3,7 +3,12 @@ namespace: TestModel
 cpp:
   sourcesOutputDir: ../../cpp/test/generated
   internalGenerateMocks: true
+  internalGenerateTranslator: true
```

Comment:
I am wondering if this should be an internal thing? Should we generate this always?

Reply:
Keeping this internal for now. We can see how generally applicable it is, and if so, what form it should take.